app/database_etl/etl_main.py

```diff
@@ -88,6 +88,18 @@ def main():
         # Create research source based on dashboard source
         research_source, research_source_cols = create_research_source_df(dashboard_source)
 
+        # remove state names from city name column and place them in their own column
+        multi_select_tables_dict["city"]["state_name"] = multi_select_tables_dict["city"]["city_name"] \
```

Comment:
Why do we need to do this in the first instance? What is it about our data structure that makes this necessary? Seems a bit convoluted.

Reply:
@rahularoradfs 2 reasons:

1. Able to get better results when querying for {city},{state} as opposed to {city}
2. Needed to ensure that the city_name column was unique so that the logic we wrote to construct bridge tables would work properly (see https://github.com/serotracker/iit-backend/blob/master/app/database_etl/postgres_tables_handler/table_generator.py#L191)

Honestly this probably deserves a refactor. If our city table contained a city_name and state_name column from the start and we modified our bridge table construction logic a bit, it would be much cleaner to read. If it’s okay I’d rather do this in a separate PR so that we can push incremental changes out.

Reply:
agree on intended change, agree on sep PR :) would love to merge this one ASAP as there are large user implications of this, incl mali and south sudan studies showing up